civic_europe/templates/formtools/wizard/wizard_form.html

```diff
@@ -46,6 +47,11 @@ <h1>{{ view.title }} | {{ form.section_name }}</h1>
 
         <div class="row">
             <div class="col-md-12">
+
+              {% if wizard.steps.prev and wizard.steps.next or wizard.steps.next %}
```

Comment:
This is the same as just `{% wizard.steps.next %}` :nerd_face:

Reply:
danke! :)